[2025-08-19T08:59:18.373+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-19T08:59:18.413+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: youtube_pipeline_batch_data.drop_views manual__2025-08-19T08:59:11.612500+00:00 [queued]>
[2025-08-19T08:59:18.425+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: youtube_pipeline_batch_data.drop_views manual__2025-08-19T08:59:11.612500+00:00 [queued]>
[2025-08-19T08:59:18.426+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 4
[2025-08-19T08:59:18.449+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): drop_views> on 2025-08-19 08:59:11.612500+00:00
[2025-08-19T08:59:18.455+0000] {standard_task_runner.py:63} INFO - Started process 2161 to run task
[2025-08-19T08:59:18.458+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'youtube_pipeline_batch_data', 'drop_views', 'manual__2025-08-19T08:59:11.612500+00:00', '--job-id', '559', '--raw', '--subdir', 'DAGS_FOLDER/youtube_pipeline.py', '--cfg-path', '/tmp/tmpvlkivjfe']
[2025-08-19T08:59:18.461+0000] {standard_task_runner.py:91} INFO - Job 559: Subtask drop_views
[2025-08-19T08:59:18.517+0000] {task_command.py:426} INFO - Running <TaskInstance: youtube_pipeline_batch_data.drop_views manual__2025-08-19T08:59:11.612500+00:00 [running]> on host 0121d0ff2e32
[2025-08-19T08:59:18.642+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='youtube_pipeline_batch_data' AIRFLOW_CTX_TASK_ID='drop_views' AIRFLOW_CTX_EXECUTION_DATE='2025-08-19T08:59:11.612500+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-19T08:59:11.612500+00:00'
[2025-08-19T08:59:18.643+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-19T08:59:18.665+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-08-19T08:59:18.666+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'docker exec youtube-pipeline-project-postgres-1 psql -U citus -d youtube_dw -c "DROP VIEW IF EXISTS avg_views_comments CASCADE; DROP VIEW IF EXISTS vw_q02_top_engagement_channels CASCADE; DROP VIEW IF EXISTS vw_q03_gold_combinations CASCADE; DROP VIEW IF EXISTS vw_q04_problem_types CASCADE; DROP VIEW IF EXISTS vw_q05_tags_core_metrics CASCADE; DROP VIEW IF EXISTS vw_q06_magic_tags CASCADE; DROP VIEW IF EXISTS vw_q07_speed_to_viral CASCADE; DROP VIEW IF EXISTS vw_q08_desc_thumb_performance CASCADE; DROP VIEW IF EXISTS vw_q09_seasonal_launch CASCADE; DROP VIEW IF EXISTS vw_q10_channel_top_video CASCADE;"']
[2025-08-19T08:59:18.681+0000] {subprocess.py:86} INFO - Output:
[2025-08-19T08:59:18.816+0000] {subprocess.py:93} INFO - psql: error: connection to server on socket "/var/run/postgresql/.s.PGSQL.5432" failed: FATAL:  role "citus" does not exist
[2025-08-19T08:59:18.823+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2025-08-19T08:59:18.824+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-19T08:59:18.846+0000] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/bash.py", line 243, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2025-08-19T08:59:18.853+0000] {taskinstance.py:1205} INFO - Marking task as UP_FOR_RETRY. dag_id=youtube_pipeline_batch_data, task_id=drop_views, execution_date=20250819T085911, start_date=20250819T085918, end_date=20250819T085918
[2025-08-19T08:59:18.873+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 559 for task drop_views (Bash command failed. The command returned a non-zero exit code 2.; 2161)
[2025-08-19T08:59:18.913+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-08-19T08:59:18.956+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-08-19T08:59:18.964+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
